docs/specification/auth/credential.md

```diff
@@ -0,0 +1,119 @@
+---
+title: Credential Authentication
+type: docs
+weight: 52
+---
+
+{{< callout type="info" >}}
+**Protocol Revision**: {{< param protocolRevision >}}
+{{< /callout >}}
+
+The Model Context Protocol (MCP) supports credential-based authentication for scenarios requiring API keys or similar secrets.
+
+## Capabilities
+
+Clients supporting credential authentication **MUST** declare it during initialization:
+
+```json
+{
+  "capabilities": {
+    "auth": {
+      "credential": true
+    }
+  }
+}
+```
+
+Servers supporting credentials **MUST** include their capabilities:
+
+```json
+{
+  "capabilities": {
+    "auth": {
+      "credential": {
+        "list": true
+      }
+    }
+  }
+}
+```
+
+## Protocol Messages
+
+### Credential Requirements
+
+Servers can list required credentials using the credential/list capability:
```

Comment:
Is this meant to be s/capability/method/? If so, can we provide an example of that request?